tests/test_calibrate.py

```diff
@@ -170,7 +170,9 @@ def _checkDoRefcats(self, doAstrometry, doPhotoCal, ids):
         config.doAstrometry = doAstrometry
         config.doPhotoCal = doPhotoCal
         config.connections.photoRefCat = "cal_ref_cat"
+        config.photoRefObjLoader.ref_dataset_name = "cal_ref_cat"
```

Comment:
Why is this necessary?  Isn't this usage deprecated?

Reply:
I think until I merge [DM-35871](https://jira.lsstcorp.org/browse/DM-35871) we might need still need this for how colorterms are used. I'll deal with it on that ticket.